### PR TITLE
[cilium][docs] add visualizations for EgressGateway modes

### PR DESCRIPTION
**Pull request by Beastlex**
## Description
Add visualizations for EgressGateway with use cases:
* base
* with VirtualIP

## Why do we need it, and what problem does it solve?
Need to make documentation and use cases available to customers

## Checklist
- [x] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: cilium
type: chore
summary: Add visualizations for EgressGateway.
impact_level: low
```

### CODE REVIEW

File: modules/021-cni-cilium/docs/README.md [hunk 1]
```diff
@@ -48,3 +48,17 @@ If you change the Cilium operating mode (the [tunnelMode](configuration.html#par
 ## A note about disabling the kube-proxy module
 
 Cilium has the same functionality as the `kube-proxy` module, so the latter is automatically disabled when the `cni-cilium` module is enabled.
+
+## A note about Egress Gateway
+
+{% alert level="warning" %} Feature is only available in Enterprise Edition {% endalert %}
+
+Basic mode. Nodes with configured IP addresses are used.
```
Comment (apolovov) [inline modules/021-cni-cilium/docs/README.md:56]:
```suggestion
### Basic mode

Using pre-configured public IPs of egress-gateway nodes.
```

File: modules/021-cni-cilium/docs/README.md [hunk 2]
```diff
@@ -48,3 +48,17 @@ If you change the Cilium operating mode (the [tunnelMode](configuration.html#par
...
+
+<div data-presentation="../../presentations/021-cni-cilium/egressgateway_base_en.pdf"></div>
+<!--- Source: https://docs.google.com/presentation/d/1Gp8b82WQQnYr6te_zBROKnKmBicdhtX4SXNXDh3lB6Q/ --->
+
+Mode with Virtual IP. Allows you to dynamically assign additional IP addresses to nodes.
```
Comment (apolovov) [inline modules/021-cni-cilium/docs/README.md:61]:
```suggestion
### Virtual IP mode

Allows you to dynamically assign additional IP addresses to nodes.
```

File: modules/021-cni-cilium/docs/README_RU.md [hunk 3]
```diff
@@ -50,3 +50,17 @@ description: Модуль cni-cilium Deckhouse обеспечивает рабо
 ## Заметка о выключении модуля kube-proxy
 
 Cilium полностью заменяет собой функционал модуля kube-proxy, поэтому тот автоматически отключается при включении модуля cni-cilium.
+
+## Заметка о Egress Gateway
+
+{% alert level="warning" %} Функция доступна только в Enterprise Edition {% endalert %}
+
+Базовый режим. Используются узлы с настроенными IP-адресами.
```
Comment (apolovov) [inline modules/021-cni-cilium/docs/README_RU.md:58]:
```suggestion
### Базовый режим

Используются предварительно настроенные IP-адреса на egress-узлах.
```

File: modules/021-cni-cilium/docs/README_RU.md [hunk 4]
```diff
@@ -50,3 +50,17 @@ description: Модуль cni-cilium Deckhouse обеспечивает рабо
...
+
+<div data-presentation="../../presentations/021-cni-cilium/egressgateway_base_ru.pdf"></div>
+<!--- Source: https://docs.google.com/presentation/d/12l4w9ZS3Hpax1B7eOptm2dQX55VVAFzRTtyihw4Ie0c/ --->
+
+Режим с Virtual IP. Позволяет динамически назначать дополнительные IP-адреса узлам.
```
Comment (apolovov) [inline modules/021-cni-cilium/docs/README_RU.md:63]:
```suggestion
### Режим с Virtual IP

Позволяет динамически назначать дополнительные IP-адреса узлам.
```

**Review by apolovov — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/stage'...

File: modules/021-cni-cilium/docs/README.md [hunk 5]
```diff
@@ -48,3 +48,21 @@ If you change the Cilium operating mode (the [tunnelMode](configuration.html#par
 ## A note about disabling the kube-proxy module
 
 Cilium has the same functionality as the `kube-proxy` module, so the latter is automatically disabled when the `cni-cilium` module is enabled.
+
+## A note about Egress Gateway
```
Comment (apolovov) [inline modules/021-cni-cilium/docs/README.md:52]:
```suggestion
## A note about fault-tolerant Egress Gateway
```

File: modules/021-cni-cilium/docs/README_RU.md [hunk 6]
```diff
@@ -50,3 +50,21 @@ description: Модуль cni-cilium Deckhouse обеспечивает рабо
 ## Заметка о выключении модуля kube-proxy
 
 Cilium полностью заменяет собой функционал модуля kube-proxy, поэтому тот автоматически отключается при включении модуля cni-cilium.
+
+## Заметка о Egress Gateway
```
Comment (apolovov) [inline modules/021-cni-cilium/docs/README_RU.md:54]:
```suggestion
## Заметка об отказоустойчивом Egress Gateway
```

**Review by apolovov — APPROVED** [review]

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [9190](https://github.com/deckhouse/deckhouse/pull/9190) to the branch [release-1.62](https://github.com/deckhouse/deckhouse/tree/release-1.62) successful!

**Pull request merged**